### PR TITLE
Updates for addon.xml.in

### PR DESCRIPTION
**Pull request by garbear**
* Adds language tags for `en_GB`
* Converts `<noicon>` and `<nofanart>` to `<assets>` tag

### CODE REVIEW

File: peripheral.joystick/addon.xml.in [hunk 1]
```diff
@@ -11,10 +11,10 @@
...
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">Kodi Joystick Library</summary>
-    <description lang="en">This library provides joystick drivers and button maps. Multiple joystick APIs are supported, including DirectX, XInput, SDL and the Linux Joystick API.</description>
+    <summary lang="en_GB">Kodi Joystick Library</summary>
+    <description lang="en_GB">This library provides joystick drivers and button maps. Multiple joystick APIs are supported, including DirectX, XInput, SDL and the Linux Joystick API.</description>
     <platform>@PLATFORM@</platform>
-    <noicon>true</noicon>
-    <nofanart>true</nofanart>
+    <assets>
+    </assets>
```
Comment (Rechi) [inline peripheral.joystick/addon.xml.in]: Why was noicon and nofanart even added initially? Only repo generator is allowed to add those (if the add-on would be shipped in a repo)
I also see no reason to add an empty assets tag.

Comment (garbear) [reply peripheral.joystick/addon.xml.in]: It prevents Kodi from hammering the server for icons and fanart. By default, Kodi tries to retrieve icon and fanart from the repo.

Comment (Rechi) [reply peripheral.joystick/addon.xml.in]: If it doesn't come from a repo, which requests does it send?
I know what it does in a repo, but that's why the repo generator adds those tags and not the user.

Comment (garbear) [reply peripheral.joystick/addon.xml.in]: I can't test Kodi on linux right now so I'm not sure. Should `<assets>` be removed?

Comment (Rechi) [reply peripheral.joystick/addon.xml.in]: assets, noicon and nofanart should be removed

Comment (garbear) [reply peripheral.joystick/addon.xml.in]: done

**Pull request merged**